### DISCUSSION TITLE
Cannot create a new repo with a template repo with a larger file (> 10MB)

### DISCUSSION THREAD

**Discussion by tnir** [community/community#22228] [New to GitHub]
<p>We cannot use a template, which contains a file with 10MB of its file size, resulting in the following:</p>
<blockquote>
<p>We’re sorry, something went wrong.<br>
We were unable to clone {template_orgname}/{template_reponame}'s contents into  <code>{new_username,new_orgname}/{new_reponame}</code> . The template you used includes files that are larger than 10 megabytes. Please ask {template_orgname} to remove those files from the template and try again.</p>
</blockquote>
<p>As a limit per individual file is 100MB (cf. <a href="https://docs.github.com/en/free-pro-team@latest/github/managing-large-files/what-is-my-disk-quota" rel="noopener nofollow ugc">https://docs.github.com/en/free-pro-team@latest/github/managing-large-files/what-is-my-disk-quota</a>), it should be the same on starting a repo with a template.</p>
<p>Example failures in various template repos:</p>
<ul>
<li><a href="https://github.com/education/classroom/issues/2521" rel="noopener nofollow ugc">https://github.com/education/classroom/issues/2521</a></li>
<li><a href="https://github.com/psmathur/fastai2-vision-app/issues/1" rel="noopener nofollow ugc">https://github.com/psmathur/fastai2-vision-app/issues/1</a></li>
<li><a href="https://github.com/johanbrandhorst/grpc-gateway-boilerplate/issues/19" rel="noopener nofollow ugc">https://github.com/johanbrandhorst/grpc-gateway-boilerplate/issues/19</a></li>
</ul>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/tnir">@tnir</a> :wave:</p>
<p>This is a great observation! Unfortunately, we do not support template repos with files greater than 10MB right now. You will need to remove those files from the template.</p>
<p>We’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.<br>
You can do so by <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request.</p>
<p>You can also keep an eye on our <a href="https://github.blog/changelog/">Changelog</a> as well as the <a href="https://github.com/github/roadmap">GitHub public roadmap</a> to see features we are working on.</p>
